packages/core/admin/admin/src/components/GuidedTour/init.js

```diff
@@ -0,0 +1,25 @@
+import set from 'lodash/set';
+import persistStateToLocaleStorage, {
+  COMPLETED_STEPS,
+  CURRENT_STEP,
+} from './utils/persistStateToLocaleStorage';
+
+const init = initialState => {
+  const guidedTourLocaleStorage = persistStateToLocaleStorage.get(COMPLETED_STEPS);
+  const currentStepLocaleStorage = persistStateToLocaleStorage.get(CURRENT_STEP);
+
+  if (guidedTourLocaleStorage) {
+    guidedTourLocaleStorage.forEach(step => {
+      const [pluginName, completedStepName] = step.split('.');
+      set(initialState, ['guidedTourState', pluginName, completedStepName], true);
```

Comment:
the set function of lodash will mutate the initialState object, is it what we want?